Example/Stripe iOS Example (Simple)/MyAPIClient.swift

```diff
@@ -21,7 +21,44 @@ class MyAPIClient: NSObject, STPBackendAPIAdapter {
             fatalError()
         }
     }
+    
+    func login(_ email: String, password: String, completion: @escaping STPErrorBlock) {
+        let url = self.baseURL.appendingPathComponent("customer/login")
+        Alamofire.request(url, method: .post, parameters: [
+            "email": email,
+            "password": password])
+            .validate(statusCode:200..<300)
+            .responseString { response in
+                switch response.result {
+                case .success:
+                    completion(nil)
+                case .failure(let error):
+                    completion(error)
+                }
+        }
+    }
 
+    func createUser(email: String, password: String, firstName: String?, lastName: String?, completion: @escaping STPErrorBlock) {
+        let url = self.baseURL.appendingPathComponent("customer")
+        var parameters: [String: String] = ["email": email, "password": password ]
+        if let firstName = firstName, !firstName.isEmpty {
+            parameters["firstName"] = firstName
+        }
+        if let lastName = lastName, !lastName.isEmpty {
+            parameters["lastName"] = lastName
+        }
+        Alamofire.request(url, method: .post, parameters: parameters)
+            .validate(statusCode:200..<300)
+            .responseString { response in
```

Comment:
Need to pass back the actual error from the server here, not just the AF one which is pretty generic.